### PR TITLE
Cleanup of generated clients

### PR DESCRIPTION
**Pull request by michaelbausor**
- Insert newlines
- Add comments
- Remove empty page streaming descriptors

### CODE REVIEW

File: src/main/resources/com/google/api/codegen/php/main.snip [hunk 1]
```diff
@@ -286,7 +297,7 @@
      *                              Google\Auth library.
      * }
      @if xapiClass.hasMissingDefaultOptions
-         * @throws ValidationException Throws a ValidationException if required arguments are missing
+         * @@throws ValidationException Throws a ValidationException if required arguments are missing
```
Comment (shinfan) [inline src/main/resources/com/google/api/codegen/php/main.snip]: Is there anyway to catch this in the baseline?

Comment (michaelbausor) [inline src/main/resources/com/google/api/codegen/php/main.snip]: The only way would be to create a new service that didn't have default service address/scopes. Maybe this is worth it? I don't know if other languages will also benefit.

Comment (shinfan) [inline src/main/resources/com/google/api/codegen/php/main.snip]: Ah this is fine. I don't think it would worth the effort.

**Comment by shinfan** [comment]
LGTM

**Pull request merged**